codegen/templates/tchannel_client.tmpl

```diff
@@ -143,6 +143,17 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		{{ end -}}
 	}
 
+	//get mapping of client method and it's timeout
+	//if mapping is not provided, use client's timeout for all the methods
+	clientMethodTimeoutMapping := make(map[string]int64)
```

Comment:
why did we decide to make the values `int64` here? If we used an `int` instead, we could get rid of the type conversion inside `configureCircuitBreaker`

Reply:
**int** doesn't for `deps.Default.Config.MustGetStruct("clients.{{$clientID}}.methodTimeoutMapping", &clientMethodTimeoutMapping)` statement, hence had to update it int64